tests/parser/EclipseGridTests.cpp

```diff
@@ -255,7 +255,7 @@ static Opm::Deck createMinpvDefaultCPDeck() {
         "  1000*1 / \n"
         "MINPV \n"
         "  / \n"
-        "MINPVFIL \n"
+        "MINPORV \n"
```

Comment:
This will throw in line 846, because the combination of MINPV and MINPORV is not allowed.